cfme/infrastructure/provider/widgetastic_views.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could this be in cloud_infra_common, since both infra and cloud providers will have the same toolbar buttons? Any buttons unique to infra/cloud could be defined here?